### ISSUE TITLE
What about max width?

### ISSUE THREAD

**Issue by OnkelTem** [doitian/obsidian-min-width#2]
Thanks for the plugin!

I understand the importance of minimum width, however I find that setting max width is as important. I don't like that my notes spread to the full width of my monitor, I always feel like a cat watching a tennis game 🤣

In other words there should be just one setting: width. I don't know what this obvious thing is not in the base functionality of Obsidian itself. I hope it can be done using plugins.

**Comment by doitian** [comment]
For width, https://github.com/deathau/sliding-panes-obsidian should work.

**Comment by OnkelTem** [comment]
@doitian yeah, thanks, that's what I use myself. But once again, it doesn't have max width setting. Which makes texts to jump and re-layout. So I thought that maybe this plugin could do that instead, and that I could have both of them playing well :)

**Comment by doitian** [comment]
I do not know how to handle the ugly gaps after setting `width` or `max-width`. You'd better use a window management tool to resize your obsidian to proper width.

![screenshot-2022101618-rqoDDwbp@2x](https://user-images.githubusercontent.com/35768/196031584-12d9b4ed-312f-4d11-ba66-5f81a1e286e8.png)

**Comment by OnkelTem** [comment]
Ugly gaps? Well, okay. But it's still much better than things like this:

![image](https://user-images.githubusercontent.com/114060/196034647-51c0d746-25da-4fa3-9409-8c0334f2e026.png)

which just renders text barely readable (line is too long).

Also, note the bold selection in the text which is in the middle of the page and which indicates the place I was reading the text. Now if I open another note, that guy just jumps to the next line, and I lose the position where I was reading:

![image](https://user-images.githubusercontent.com/114060/196034737-faa78a13-3968-4d7e-9baf-34228ff3eb46.png)

So I was talking about Obsidian issue with text layout inconsistency. Which is of much higher priority than "ugly gaps"

**Comment by doitian** [comment]
There's an option `Editor ➤ Display ➤ Readable line length`. The default line length is `40rem`, which can be customized by theme or css snippet:

```
body {
  --line-width: 42rem;
}
```
